website/docs/reference/public-signup.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Public Invite Links
+---
+
+Public invite links allow you to invite team members to unleash very quickly and efficiently.
+
+## Public signup tokens
+
+You can use the form to create a new _public signup token_ and a new invite link will be created for you and displayed in the admin UI
+
+Copy and share this link with anyone you would like to give access.
+
+Anyone signing up with Unleash using the _invite-link_
+
+### Creating a new token
+
+- Go to settings -> User
+
+![Go to settings -> User ](/img/public-signup-step1.png)
+
+
+- Click the Create Invite Link button
+
+![Click the Create Invite Link button](/img/public-signup-step2.png)
+
+- Set an expiration for your new token
+
+![Create invite link form](/img/public-signup-step3-create_link.png)
+
+**`expireAt`**: Public signup tokens will expire on this date
+
+- Token created - Link will be available in your Users page (until you delete it)
+
+![Link Created](/img/public-signup-step4_link_Created.png)
+
+
+Once your link is created, you can copy and share with anyone you like.
+
+### Updating/Deleting a token
+- You can use the same form to update or delete your token
+![Update invite link form](/img/public-signup-step3-create_link.png)
+
+## Invite links
+Any user using the _invite-link_ will be allowed to signup with Unleash using the UI and will be assigned the **_Viewer_** role.
+
+The user will be taken directly to the signup page, where they can fill in their data and create an account.
+
+![User signup form](/img/public-invite_signup.png)
+
+Once the user is logged in as a **_Viewer_**, they will be able to navigate the Unleash UI but without being able to edit or create anything
+![User signup form](/img/public-invite_step2_signed_in.png)
```

Comment:
Nice work here! 🙌🏼  However, it mixes how-to documentation with reference documentation, which is something we try to avoid (see the [diataxis framework docs](https://diataxis.fr/) for more on the documentation categories we use).

Instead I suggest the following: 
1. Keep the reference content in this doc. I've added a suggestion that trims away the how-to guide and adds a little more reference content. Please review it to make sure it's correct.
2. Rename this doc to `public-invite-links.mdx` instead (for clarity and keeping the title the same).
3. Create a new file `website/docs/how-to/how-to-manage-public-invite-tokens.mdx` and move the how-to content in there. I'll add a code snippet with the suggested content for that file at the end of this comment.

## Reference suggestion:

```suggestion
Public invite links let you invite team members to your Unleash instance. Any user with an invite link can sign up to Unleash instance that created the link. The user will get the **viewer** role (refer to the [_standard roles_ section of the RBAC document](../user_guide/rbac.md#standard-roles) for more information on roles).

User who follow the invite link are taken directly to the Unleash sign-up page, where they can create an account.

Only **Unleash instance admins** can create public invite links.

![An Unleash signup form for new users](/img/public-invite_signup.png)

## Public sign-up tokens

The most important part of a public sign-up link is the sign-up token. The token is added as the `invite` query parameter to the invite link.

Each token has an **expiry date**. After this expiry date, the token will stop working and users can no longer sign up using an invite link with that token.

## Creating, updating, and deleting tokens

You can [create, update and delete tokens via the Unleash Admin UI](../how-to/how-to-manage-public-invite-tokens.mdx) or via the [Unleash API](../reference/api/unleash/public-signup-tokens.api.md "Public sign-up tokens API documentation").

A token is active as soon as it's created and stops working as soon as it's deleted or expired.

You can only have one active invite token at a time. If you already have an active token, you must delete it to create a new one.
```

## How-to suggestion

```md
---
title: How to manage public invite tokens
---

[Public invite links](../reference/public-invite-links.mdx) let you invite new members to an Unleash instance. A key part of an invite link is the public invite token. This guide shows you how to use the Unleash admin UI to create, update, and delete public invite tokens. You can also [manage public signup tokens via the Unleash API](../reference/api/unleash/public-signup-tokens.api.md).

Only Unleash instance admins have the necessary permissions to create and manage public invite tokens.

## Creating a token

1. Navigate to the **users** page in Unleash and use the **create invite link** button

	![The settings menu in the Unleash nav bar with the "users" link highlighted.](/img/public-signup-step1.png)

	![The Unleash users page. There is a separate "create invite link" section above the list of users.](/img/public-signup-step2.png)

2. Fill out the "create invite link" form and (optionally) copy the invite link. You can always get the link later.
   ![A short form with only one field: token expiry.](/img/public-signup-step3-create_link.png)

	![An "invite link created" modal. It contains an invite link that can be copied and some info on how to use it.](/img/public-signup-step4_link_Created.png)

## Updating/Deleting a token

1. Follow the steps in [the previous paragraph](#creating-a-token) to navigate to the users page. 
2. When you have an active invite token, use the button labeled "update invite link". 
3. Use the form to edit the expiry for the token or to delete it entirely.
```